evm/utils/fixture_tests.py

```diff
@@ -205,8 +206,14 @@ def normalize_to_address(value):
 
 def robust_decode_hex(value):
     unprefixed_value = remove_0x_prefix(value)
-    if len(unprefixed_value) % 2:
-        return decode_hex(pad_left(unprefixed_value, len(unprefixed_value) + 1, b'0'))
+
+    if is_bytes(value):
+        text_value = codecs.decode(unprefixed_value, 'utf8')
```

Comment:
_(if I'm wrong about the whole method being replaceable by `hexstr_if_str(to_bytes)`)_

I don't think we want to decode to text using utf-8 here. Guessing from the function context, I think we would want to convert to a hex string.

Reply:
you were correct, was able to replace the whole thing.